server/server.js

```diff
@@ -22,19 +22,51 @@ app.get('/reviews/:productId', (req, res) => {
   });
 });
 
-// endpoint for reviews data to return rating for different modules
-app.get('/reviews/all/:productId', (req, res) => {
-  const sql = 'SELECT * FROM reviews WHERE product_id = ?';
-  db.query(sql, req.params.productId, (err, reviews) => {
+// initial store data (need it only once, doesn't change when tabs or pager will be clicked)
+app.get('/review-summary/:productId', (req, res) => {
+  const sql = 'SELECT COUNT(*) as count, AVG(rating) as avgRating FROM reviews LEFT JOIN product_to_stores ON reviews.product_id = product_to_stores.id WHERE store_id = (SELECT store_id FROM product_to_stores WHERE id = ?); SELECT COUNT(*) as count FROM reviews WHERE product_id = ?;';
```

Comment:
Might make this file easier to read if you broke out the routes into their own folders/files and then imported the routes here. Low priority for FEC.

Reply:
Hm, interesting, I've not seen that before. I don't see the point here but keep in it mind for larger projects.